### DISCUSSION TITLE
Server Set-up

### DISCUSSION THREAD

**Discussion by BiggHoss** [official-antistasi-community/A3-Antistasi#2595] [Q&A] [answered]
I'm trying to set up a server using FASTER. With the new 3.0 mod where do I find the mission file?

**Comment by Bob-Murphy** [comment, marked as answer]
The missions are within the mod.
You can find the mod either on steam or here https://github.com/official-antistasi-community/A3-Antistasi/releases/tag/3.0.0
Just load the mod properly as you would load every other mod, select a compatible map and select the mission called "Antistasi Community 3.0.0".
